src/sagemaker/image_uri_config/xgboost.json

```diff
@@ -1,7 +1,9 @@
 {
     "scope": ["inference", "training"],
     "version_aliases": {
-        "latest": "1"
+        "latest": "1", 
+        "0.90": "0.90-2", 
+        "1.0": "1.0-1" 
     },
     "versions": {
```

Comment:
When you update version_aliases, you will also need to update versions with the corresponding regional accounts, py_versions, repository name etc.

Reply:
I'm confused on this point. I interpret the `version_aliases` as a means to allow users type the alias instead of the full version (e.g input `1.0` instead of `1.0-1`). Adding a new alias should not affect the destination version number, isn't it? In this case, `0.90-2` and `1.0-1` remain the same as before.